src/features/timetable/lib/src/timetable_screen.dart

```diff
@@ -0,0 +1,163 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import 'package:timetable/src/bl/abstractions/text_localizer.dart';
+import 'package:timetable/src/components/timetable_tab.dart';
+import 'package:timetable/src/bl/bloc/timetable_bloc.dart';
+import 'package:timetable/src/bl/abstractions/timetable_loader.dart';
+import 'package:timetable/src/bl/models/models.dart';
+
+import '../timetable.dart';
+
+class TimetableScreen extends StatefulWidget {
+  final TextLocalizer textLocalizer;
+  final TimetableLoader timetableLoader;
+
+  TimetableScreen({required this.textLocalizer, required this.timetableLoader});
+
+  @override
+  _TimetableScreenState createState() => _TimetableScreenState();
+}
+
+class _TimetableScreenState extends State<TimetableScreen> {
+  MediaQueryData get mediaQuery => MediaQuery.of(context);
+  int initialIndex = (DateTime.now().weekday - 1) % 6;
+
+  late TimetableBloc timetableBloc;
```

Comment:
init values first

Reply:
we can`t get parameter from widget in initializer